### ISSUE TITLE
Setup Local

### ISSUE THREAD

**Issue by ChrisBup** [RadishLab/TAT001#30]
@andyinabox @jaymullen When running `cap staging utils:sync_db`, I'm running into this error:

![screen shot 2017-12-04 at 11 16 39 am](https://user-images.githubusercontent.com/7212504/33563295-8c16d9ae-d8e5-11e7-800f-8889c3f01849.png)

Troubleshooting it now but let me know if you know how I should resolve.

**Comment by jaymullen** [comment]
@ChrisBup Did you try running `vagrant plugin install`?

**Comment by ChrisBup** [comment]
@jaymullen Yeah but I get another error. Figuring out how to invoke it properly...
![screen shot 2017-12-04 at 11 30 14 am](https://user-images.githubusercontent.com/7212504/33563698-a05f393c-d8e6-11e7-8765-891f53911cd9.png)

**Comment by andyinabox** [comment]
@ChrisBup I'm not having any trouble, which I'm guessing means that I already had the required plugin installed. I'm not sure how to figure out which ones are causing your error, but here's the list of vagrant plugins installed in my local env:

```
vagrant-bindfs (1.0.1)
vagrant-hostmanager (1.8.5)
vagrant-hostsupdater (1.0.2)
vagrant-share (1.1.6, system)
vagrant-triggers (0.5.3)
vagrant-vbguest (0.13.0)
```

@jaymullen do you know which vagrant plugins we need for this stack?

**Comment by ChrisBup** [comment]
For reference, it seems to have been two separate issues. Vagrant was requiring plugins I didn't have so I added them:
```
vagrant-bindfs (1.0.10)
vagrant-hostmanager (1.8.7)
vagrant-hostsupdater (1.0.2)
vagrant-librarian-chef-nochef (0.2.0)
vagrant-share (1.1.6)
vagrant-triggers (0.5.3)
vagrant-vbguest (0.15.0)
```
and restarting my machine resolved it. There's also an issue with syncing the database which may be related to the High Sierra root bug update but still working on it. @andyinabox , you can run `cap staging utils:db` fine, right? Also, what version of ruby are you using?

**Comment by andyinabox** [comment]
@ChrisBup  ah that makes sense, I'm still on El Capitan. Yes I can run `cap staging utils:sync_db` fine (is that what you meant?). Here's what I'm running:

```
ruby 2.1.2p95 (2014-05-08 revision 45877) [x86_64-darwin13.0]

Vagrant 1.9.1

Capistrano Version: 3.7.1 (Rake Version: 12.3.0)
```

**Comment by ChrisBup** [comment]
@andyinabox thanks, yeah that's what I meant.

I'm running:
```
ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-darwin16]
Vagrant 2.0.1
Capistrano 3.10.0 (Rake Version: 12.3.0)
```
I think I'm going to start the process over and see if I can troubleshoot that way.

**Comment by ChrisBup** [comment]
Okay, I started the process over and reprovisioned vagrant again but `vagrant up` went back to failing on `TASK [php-fpm : PHP-FPM | install]`

I was running PHP version 7.1.3 so I manually updated to 7.2.0 but same thing:

![screen shot 2017-12-05 at 11 23 56 am](https://user-images.githubusercontent.com/7212504/33617884-f9f1f4a8-d9ae-11e7-872a-cc71b2ce679c.png)

`cap staging utils:sync_db` is now working but a `tat001.alloy.radish.loc` WordPress directory is not showing up inside of the TAT001 directory. And http://tat001.alloy.radish.loc/ shows an nginx welcome screen:

<img width="725" alt="screen shot 2017-12-05 at 11 28 06 am" src="https://user-images.githubusercontent.com/7212504/33618019-66c26b08-d9af-11e7-8f1d-d564313b284b.png">

Troubleshooting that `php-fpm` bug now.

**Comment by ChrisBup** [comment]
@andyinabox @jaymullen @edwerd  I think it's fixed. I switched to `ruby 2.1.10p492 (2016-04-01 revision 54464) [x86_64-darwin17.0]` from `ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-darwin16]` That got the dev environment looking right. Then, I had to reinstall Capistrano and that allowed the cap commands to work again.

Another weird thing is that different ruby versions are showing in different panels within the same tmux session:

![screen shot 2017-12-05 at 1 36 08 pm](https://user-images.githubusercontent.com/7212504/33624208-9646fb84-d9c1-11e7-97d1-b9303d303e84.png)

Not sure how much High Sierra is to blame for all this but it might also have to do with RVM which I'm using to manage ruby versions: https://github.com/tpope/vim-dispatch/issues/9 but I'm not going to mess with it. Maybe I'll switch to rbenv next time: https://github.com/rbenv/rbenv/wiki/Why-rbenv%3F
